src/app/pages/create-wallet/create-wallet.tsx

```diff
@@ -1,7 +1,158 @@
+                                {state.words[2]}
+                            </LayoutGrid.Cell>
+                        </LayoutGrid.Inner>
+                        <LayoutGrid.Inner>
+                            <LayoutGrid.Cell cols={4} className="secret-word">
+                                {state.words[3]}
+                            </LayoutGrid.Cell>
+                            <LayoutGrid.Cell cols={4} className="secret-word">
+                                {state.words[4]}
+                            </LayoutGrid.Cell>
+                            <LayoutGrid.Cell cols={4} className="secret-word">
+                                {state.words[5]}
+                            </LayoutGrid.Cell>
+                        </LayoutGrid.Inner>
+                        <LayoutGrid.Inner>
+                            <LayoutGrid.Cell cols={4} className="secret-word">
+                                {state.words[6]}
+                            </LayoutGrid.Cell>
+                            <LayoutGrid.Cell cols={4} className="secret-word">
+                                {state.words[7]}
+                            </LayoutGrid.Cell>
+                            <LayoutGrid.Cell cols={4} className="secret-word">
+                                {state.words[8]}
+                            </LayoutGrid.Cell>
+                        </LayoutGrid.Inner>
+                        <LayoutGrid.Inner>
+                            <LayoutGrid.Cell cols={4} className="secret-word">
+                                {state.words[9]}
+                            </LayoutGrid.Cell>
+                            <LayoutGrid.Cell cols={4} className="secret-word">
+                                {state.words[10]}
+                            </LayoutGrid.Cell>
+                            <LayoutGrid.Cell cols={4} className="secret-word">
+                                {state.words[11]}
+                            </LayoutGrid.Cell>
+                        </LayoutGrid.Inner>
+                        <LayoutGrid.Inner>
+                            <LayoutGrid.Cell cols={12}>
+                                <Card.ActionButton
+                                    ripple
+                                    className="copy-to-clipboard"
+                                    onClick={() => this.copyToClipboard()}
+                                >
+                                    COPY TO CLIPBOARD
+                                </Card.ActionButton>
+                            </LayoutGrid.Cell>
+                        </LayoutGrid.Inner>
+                    </LayoutGrid>
+                </div>
+            );
+        } else {
+            wordsList = (
+                <div
+                    id="hidden-words"
+                    onClick={() => {
+                        this.revealWords();
+                    }}
+                >
+                    <img id="eye" src="/assets/eye.png" />
```

Comment:
use material icon instead (remove_red_eye -> this is the icon from design), use this tool https://material.io/tools/icons/ to find icons.

Also, when adding a new icon please  use SVG instead of png (if possible).